vars/mtaBuild.groovy

```diff
@@ -114,3 +119,8 @@ void call(Map parameters = [:]) {
         }
     }
 }
+
+private downloadSettingsFromUrl(String url){
```

Comment:
What about moving this into a helper class as it is a code duplication with mavenExecute